uhdm-plugin/uhdmastfrontend.cc

```diff
@@ -105,10 +111,12 @@ struct UhdmAstFrontend : public Frontend {
             vpi_release_handle(design);
         bool dump_ast1 = shared.debug_flag;
         bool dump_ast2 = shared.debug_flag;
+        bool dump_vlog1 = shared.debug_flag;
+        bool dump_vlog2 = shared.debug_flag;
```

Comment:
In mainline yosys `-debug` is an alias for -dump_ast1 -dump_ast2 -dump_vlog1 -dump_vlog2 -yydebug.
I think it would be better to keep the same meaning of the options between frontends.

Reply:
Ok, so in this case we need to add separate options to dump only AST and not verilog.

Reply:
Done, in base uhdm frontend.